src/components/enterprise-page/data/service.js

```diff
@@ -22,3 +23,26 @@ export const updateUserActiveEnterprise = (enterpriseId) => {
     formData,
   );
 };
+
+async function fetchData(url, linkedEnterprises = []) {
+  let linkedEnterprisesCopy = [...linkedEnterprises];
+  const response = await getAuthenticatedHttpClient().get(url);
+  const responseData = camelCaseObject(response.data);
+  linkedEnterprisesCopy = linkedEnterprisesCopy.concat(responseData.results);
```

Comment:
nit: if you wanted to avoid redefining `linkedEnterprisesCopy`, I believe you could do the following as well:

```
const linkedEnterprisesCopy = [...linkedEnterprises];
linkedEnterprisesCopy.push(...responseData.results);
```

Reply:
love it